x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/PlannerUtils.java

```diff
@@ -296,6 +272,6 @@ public static ElementType toElementType(DataType dataType, MappedFieldType.Field
      * Returns DOC_VALUES if the given boolean is set.
      */
     public static MappedFieldType.FieldExtractPreference extractPreference(boolean hasPreference) {
-        return hasPreference ? MappedFieldType.FieldExtractPreference.DOC_VALUES : MappedFieldType.FieldExtractPreference.NONE;
+        return hasPreference ? DOC_VALUES : MappedFieldType.FieldExtractPreference.NONE;
```

Comment:
Nit: might be worth statically importing `MappedFieldType.FieldExtractPreference.NONE` as well, they're part of the same enum.